metricbeat/module/kubernetes/state_persistentvolume/_meta/test/ksm-unit-1.8

```diff
@@ -0,0 +1,37 @@
+# TYPE kube_persistentvolume_status_phase gauge
+kube_persistentvolume_status_phase{persistentvolume="test-pv-pending",phase="Available"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-pending",phase="Bound"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-pending",phase="Failed"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-pending",phase="Pending"} 1
+kube_persistentvolume_status_phase{persistentvolume="test-pv-pending",phase="Released"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-bound",phase="Available"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-bound",phase="Bound"} 1
+kube_persistentvolume_status_phase{persistentvolume="test-pv-bound",phase="Failed"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-bound",phase="Pending"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-bound",phase="Released"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-released",phase="Available"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-released",phase="Bound"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-released",phase="Failed"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-released",phase="Pending"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-released",phase="Released"} 1
+kube_persistentvolume_status_phase{persistentvolume="test-pv-failed",phase="Available"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-failed",phase="Bound"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-failed",phase="Failed"} 1
+kube_persistentvolume_status_phase{persistentvolume="test-pv-failed",phase="Pending"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-failed",phase="Released"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-available",phase="Available"} 1
+kube_persistentvolume_status_phase{persistentvolume="test-pv-available",phase="Bound"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-available",phase="Failed"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-available",phase="Pending"} 0
+kube_persistentvolume_status_phase{persistentvolume="test-pv-available",phase="Released"} 0
+# HELP kube_persistentvolume_info Information about persistentvolume.
+# TYPE kube_persistentvolume_info gauge
+kube_persistentvolume_info{persistentvolume="test-pv-available",storageclass=""} 1
+# HELP kube_persistentvolume_labels Kubernetes labels converted to Prometheus labels.
+# TYPE kube_persistentvolume_labels gauge
+kube_persistentvolume_labels{label_app="mysql-server",persistentvolume="test-labeled-pv"} 1
+kube_persistentvolume_labels{persistentvolume="test-unlabeled-pv"} 1
+# HELP kube_persistentvolume_capacity_bytes Persistentvolume capacity in bytes.
+# TYPE kube_persistentvolume_capacity_bytes gauge
+kube_persistentvolume_capacity_bytes{persistentvolume="test-pv"} 5.36870912e+09
```

Comment:
how come this one (`test-pv`) doesn't have an status phase?

Reply:
those test cases are taken from kube-state-metrics unit tests in order to test all tested cases upstream. Those are not real but created at ksm for each of the supported scenarios.

Using a (real)  resource from one of our kubernetes clusters targeted a very small subset of cases.